### ISSUE TITLE
Error highlighting not reset when fixing error in skin files

### ISSUE THREAD

**Issue by maltaisn** [BlueBoxWare/LibGDXPlugin#20]
When an error highlights a part of the code, the error only goes away when the highlighted code is completely erased. For example, say that after doing some refactoring, there are errors all over my skin file. After fixing them, they still show up, so I cut the whole file and paste it and the error highlighting goes away.

I'm using IntelliJ 2019.2 but if I recall, this has been happening since 2019.1 if not longer.

![error](https://user-images.githubusercontent.com/33690959/61998371-00a93300-b07d-11e9-8a11-b87b680633b8.gif)

**Comment by BlueBoxWare** [comment]
Hmm, I can't reproduce this. Does this also happen with other file types?

**Comment by maltaisn** [comment]
No, only with skin files. I just tried reinstalling the plugin and invalidating caches but the problem is still there.

**Comment by BlueBoxWare** [comment]
I assume this problem no longer appears. Please let me know if it still happens!
